test/hotspot/jtreg/runtime/cds/appcds/DumpClassListWithLF.java

```diff
@@ -49,41 +49,41 @@ public static void main(String[] args) throws Throwable {
             appJar, classlist(
                 "Hello",
-                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic L7_L (success)",
-                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic LL_I (success)"),
+                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic L7_L",
+                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic LL_I"),
                 MESSAGE_OK);
 
         // 2. The two lines with incorrect format of function signitures lead regeneration of holder class failed.
         dumpShouldPass(
             "TESTCASE 2: With incorrect signature",
             appJar, classlist(
                 "Hello",
-                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic L7_L-XXX (success)",
-                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic LL_I-YYY (success)"),
+                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic L7_L-XXX",
+                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeStatic LL_I-YYY"),
                 MESSAGE_NOT_OK);
-        // 3. The two lines with arbitrary invoke names is OK.
+        // 3. The two lines with arbitrary invoke names is OK, also ending with any word is OK.
         dumpShouldPass(
             "TESTCASE 3: With incorrect invoke names is OK",
             appJar, classlist(
                 "Hello",
-                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeNothing  L7_L (success)",
-                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeNothing  LL_I (success)"),
+                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeNothing  L7_L (anyword)",
+                "@lambda-form-invoker [LF_RESOLVE] java.lang.invoke.DirectMethodHandle$Holder invokeNothing  LL_I anyword"),
                 MESSAGE_OK);
         // 4. The two lines with non existed class name, since only 4 holder classes recognizable, all other names will be ignored.
         dumpShouldPass(
             "TESTCASE 4: With incorrect class name will be ignored",
             appJar, classlist(
                 "Hello",
-                "@lambda-form-invoker [LF_RESOLVE] my.nonexist.package.MyNonExistClassName$holder invokeStatic  L7_L (success)",
-                "@lambda-form-invoker [LF_RESOLVE] my.nonexist.package.MyNonExistClassName$holder invokeStatic  LL_I (success)"),
+                "@lambda-form-invoker [LF_RESOLVE] my.nonexist.package.MyNonExistClassName$holder invokeStatic  L7_L",
+                "@lambda-form-invoker [LF_RESOLVE] my.nonexist.package.MyNonExistClassName$holder invokeStatic  LL_I"),
```

Comment:
I think it's dangerous to allow arbitrary class names here. InvokerBytecodeGenerator doesn't check the classname. This will make it possible to overwrite the contents of arbitrary classes. We should have a check here and allow only the specific holder classes that are supported.